src/js/models/search/SearchAwardsOperation.js

```diff
@@ -0,0 +1,241 @@
+        this.fundingAgencies = state.selectedFundingAgencies.toArray();
+
+        this.selectedRecipients = state.selectedRecipients.toArray();
+        this.recipientDomesticForeign = state.recipientDomesticForeign;
+        this.selectedRecipientLocations = state.selectedRecipientLocations.toArray();
+        this.recipientType = state.recipientType.toArray();
+
+        this.selectedLocations = state.selectedLocations.toArray();
+        this.locationDomesticForeign = state.locationDomesticForeign;
+
+        this.awardAmounts = state.awardAmounts.toArray();
+
+        this.selectedAwardIDs = state.selectedAwardIDs.toArray();
+
+        this.selectedCFDA = state.selectedCFDA.toArray();
+        this.selectedNAICS = state.selectedNAICS.toArray();
+        this.selectedPSC = state.selectedPSC.toArray();
+
+        // TODO - Mike Bray - Enable these Other Award Filters when they're available
+        // this.pricingType = state.pricingType.toArray();
+        // this.setAside = state.setAside.toArray();
+        // this.extentCompeted = state.extentCompeted.toArray();
+    }
+
+    params() {
+        // Convert the search operation into JS objects
+        const filters = {};
+
+        // Add keyword
+        if (this.keyword !== '') {
+            filters[rootKeys.keyword] = this.keyword;
+        }
+
+        // Add Time Period
+        if (this.timePeriodFY.length > 0 || this.timePeriodRange.length === 2) {
+            if (this.timePeriodType === 'fy' && this.timePeriodFY.length > 0) {
+                filters[rootKeys.timePeriod] = this.timePeriodFY.map((fy) =>
+                    ({
+                        [timePeriodKeys.startDate]: `${fy - 1}-10-01`,
+                        [timePeriodKeys.endDate]: `${fy}-09-30`
+                    })
+                );
+            }
+            else if (this.timePeriodType === 'dr' && this.timePeriodRange.length === 2) {
+                filters[rootKeys.timePeriod] = {
+                    [timePeriodKeys.startDate]: this.timePeriodRange[0],
+                    [timePeriodKeys.endDate]: this.timePeriodRange[1]
+                };
+            }
+        }
+
+        // Add award types
+        if (this.awardType.length > 0) {
+            filters[rootKeys.awardType] = this.awardType;
+        }
+
+        // Add Agencies
+        if (this.fundingAgencies.length > 0 || this.awardingAgencies.length > 0) {
+            const agencies = [];
+
+            // Funding Agencies are toptier-only
+            this.fundingAgencies.forEach((agencyArray) => {
+                agencies.push({
+                    [agencyKeys.type]: 'funding',
+                    [agencyKeys.tier]: 'toptier',
+                    [agencyKeys.name]: agencyArray.toptier_agency.name
+                });
+            });
+
+            // Awarding Agencies can be both toptier and subtier
+            this.awardingAgencies.forEach((agencyArray) => {
+                const agencyName = agencyArray.agencyType === 'toptier'
```

Comment:
Should we do `agencyName = agencyArray[`${agencyArray.agencyType}_agency`].name` so it can handle offices in the future?